staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -349,7 +350,7 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 	}
 
 	watchCache := newWatchCache(
-		config.CacheCapacity, config.KeyFunc, cacher.processEvent, config.GetAttrsFunc, config.Versioner, config.Indexers)
+		config.CacheCapacity, config.KeyFunc, cacher.processEvent, config.GetAttrsFunc, config.Versioner, config.Indexers, objType.String())
```

Comment:
Let's pass objType instead of objType.String().

Reply:
Done.